docs/content/querying/lookups.md

```diff
@@ -290,3 +290,27 @@ To test this setup, you can send key/value pairs to a kafka stream via the follo
 `./bin/kafka-console-producer.sh --property parse.key=true --property key.separator="->" --broker-list localhost:9092 --topic testTopic`
 
 Renames can then be published as `OLD_VAL->NEW_VAL` followed by newline (enter or return)
+
+# Config examples
+
+## Per node config
+The simplest way to configure a namespace is to specify the lookup in the configuration `runtime.properties` for every node which services queries. This includes Brokers, Historicals, and Peons.
+`druid.query.extraction.namespace.lookups=[{"type":"uri","namespace":"iso_country2un_country","uri":"s3://some_bucket/some_prefix/","versionRegex":"\\\\Qcountry_code_data.csv\\\\E","namespaceParseSpec":{"format":"tsv","columns":["country_name","iso_country_code","un_country_code","un_category_name","un_category_code"],"keyColumn":"iso_country_code","valueColumn":"un_country_code"},"updateMs":600000}]`
```

Comment:
this need to be fixed, since we have to provide names  outside of the lookup thought